README.md

```diff
@@ -82,10 +82,10 @@ On macOS if you installed via `brew`
 /usr/local/Cellar/azure-functions-core-tools/<version>/workers/
 ```
 
-Under the `workers/powershell` folder, create a folder with the name `7` if it does not exist yet. Copy the result of the `publish` directory into the `workers/powershell/7` folder, and copy the `publish/worker.config.json` file into the `workers/powershell` folder:
+Under the `workers/powershell` folder, create a folder with the name `7` if it does not exist yet. Copy the result of the `publish` directory into the `workers/powershell/7.4` folder, and copy the `publish/worker.config.json` file into the `workers/powershell` folder:
```

Comment:
Should this be "create a folder with the name `7.4` if it does not exist yet"?

Reply:
If the source contains more than one file (which is the case in this scenario), then PowerShell creates the destination  folder if it does not exist, so the copy-item command part is ok. However, I can still see from references in this file to PowerShell 7, so I will fix this.